rudder-web/src/main/scala/com/normation/rudder/web/snippet/SendMetricsPopup.scala

```diff
@@ -58,7 +58,7 @@ class SendMetricsPopup extends DispatchSnippet with Loggable {
           logger.warn(msg.messageChain)
           false
         }
-        if (showPopup) {
+        if (true) {
```

Comment:
Why do you replaced by true ?  

If that is ok , then remove the if, no need to have a if(true) :)